internal/envtest/images.go

```diff
@@ -0,0 +1,41 @@
+package envtest
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/k8ssandra/k8ssandra-client/pkg/kubernetes"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func DeployImageConfig(cli client.Client) error {
```

Comment:
issue: I don't see this code being called from anywhere?

Reply:
Indeed, sorry. I think this must have come over when I copied some of the new envtest frameworks from elsewhere.